cli/src/main/scala/org/apache/celeborn/cli/common/CommonOptions.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.cli.common
+
+import picocli.CommandLine.{Command, Option, Spec}
+import picocli.CommandLine.Model.CommandSpec
+
+@Command
+class CommonOptions {
+
+  @Spec var spec: CommandSpec = _ // injected by picocli
+
+  @Option(
+    names = Array("--hostport"),
+    paramLabel = "host:port",
+    description = Array("The host and port"))
```

Comment:
This is the host and RPC port to connect for a component instance.

Reply:
it should be http port :)

Reply:
I will fix the description.